posts/dask-summit-takeaway.md

```diff
@@ -0,0 +1,143 @@
+---
+author: Max Grover
+date: 2021-5-21
+tags: dask, conference, open-science
+---
+
+# Dask Distributed Summit 2021 Takeaways
+
+## [Pangeo Session](https://zoom.us/rec/play/bVuz_sZOnzsH8fJdrsvH7T_XgXBg64cRSgbV4PERZByVgoLzUDXXoc1pq02Hcfg1oxE7uzAa919Jly1p.tmy6yMikcfgRVHi2?startTime=1621447260000&_x_zm_rtaid=P6Q_HcGvQa-lvvjzHzq4Rg.1621536889692.ea182bc019aed9fbea835c35b62d5ca5&_x_zm_rhtaid=831)
+
+- Cloud optimized datasets help improve speed of analysis
+  - Entire 4 TB datasets open up in a few seconds
+  - Important to chunk you data - store in object-store
+- Reading netcdf data remotely, especially on the cloud, can take a while...
+  - Even with using dask-delayed, quite slow
+- Open-source projects, such as XGCM, make answering big science questions easier
+  - Ex. looking at oxygen minimum zones, converting to density coords
```

Comment:
Julius' slides: https://speakerdeck.com/jbusecke/dask-and-the-ocean-death-zones-lessons-from-a-real-life-earth-science-workflow-with-a-fullish-pangeo-stack

Reply:
I included this below in the "talk links" section